src/callinput.c

```diff
@@ -147,6 +147,7 @@ int callinput(void) {
     int j, t, x = 0;
     char instring[2] = { '\0', '\0' };
     static int lastwindow;
+    extern int escape_pressed;
```

Comment:
let's add globals to `globalvars.h`. This avoids having to list all used globals separately.

Reply:
I think that's should be useful.